ktxspec.adoc

```diff
@@ -633,7 +635,8 @@ supercompression.
 | 1                    | BasisLZ         | <<etc1s_slice,ETC1S Slice Decoding>>  | <<basislz_gd,BasisLZ Global Data>>
 | 2                    | Zstandard       | <<RFC8478>>       | n/a
 | 3                    | ZLIB            | <<RFC1950>>       | n/a
-| 4･･･0xffff           | Reserved^1^     |                   |
+| 4                    | Basis GPU Photo 6x6 | https://github.com/BinomialLLC/basis_universal/wiki/ASTC-HDR-6x6-Intermediate-File-Format-(Basis-GPU-Photo-6x6)        | n/a
```

Comment:
Need to figure out what to call this. Note that this is just the supercompressed bitstream not a "file format" which the linked document name implies. @richgel999 any suggestions?

Reply:
That scheme is a bit similar to BasisLZ but does not use Huffman coding. It's a mix of RLE, VLC, and logical repackaging of ASTC fields.

Reply:
Yes - it's command based. The current format is easy for the RDO coder to target (it's trivial to accurate estimate command bit prices, and all bit prices are integer and stable). Adding entropy coding and eliminating other redundancies would probably boost compression significantly (10-30%?). 

The high-level format is documented here:
https://github.com/BinomialLLC/basis_universal/wiki/UASTC-HDR-6x6-Intermediate-File-Format-(Basis-GPU-Photo-6x6)

Adding some sort of entropy coding to the format would be a relatively easy next step.

Reply:
If entropy coding is coming, we shouldn't standardize it now.

Reply:
Entropy coding is a tradeoff, and we're still going to support the current files. Most of the compression value comes from ASTC itself and the encoder's RDO, so it'll remain optional.

Reply:
Okay. I have moved GPU Photo supercompression to the vendor list. See https://github.com/KhronosGroup/KTX-Specification/pull/216/commits/291aba1d53c3e6b047b44ae9cf9e1c5b1e28d344. This means Rich can make legal KTX2 files now. When entropy coding is finalized we can add it to the standardized list with the name minus the BINOMIAL suffix.

Reply:
> I have moved GPU Photo supercompression to the vendor list.

@MarkCallow I think this is unnecessary:
1. The scheme is now called "UASTC HDR 6x6 Intermediate" in Basis Universal readme.
2. We should be able to standardize it without RDO for now and add one more enum later if needed.

Reply:
Here is a screenshot of the formatted vendor scheme table.

<img width="1082" alt="Screenshot 2025-02-25 at 21 47 00" src="https://github.com/user-attachments/assets/a6884775-2859-4ac1-8160-fd5634fd65db" />

Reply:
> > I have moved GPU Photo supercompression to the vendor list.
> 
> @MarkCallow I think this is unnecessary:
> 
I was responding to your earlier comment

> If entropy coding is coming, we shouldn't standardize it now.

I had somehow missed Rich's comment about the software continuing to support current files and entropy coding being optional. I'll revert the change tomorrow.

> The scheme is now called "UASTC HDR 6x6 Intermediate"

Yes. I find it confusing to use the same name for both a supercompression scheme and a block-compressed format. Plus it is a very long name.

Reply:
> @richgel999 what contact info would you like to include in the vendor table?

rich@binomial.info is good.